pages/index.tsx

```diff
@@ -1,19 +1,27 @@
 import "isomorphic-fetch";
 import * as React from "react";
 
+import { Context } from "koa";
 import { ChatApp } from "../components/ChatApp";
 import { Nav } from "../components/Nav";
-import admins from "../lib/admins";
-import candidates from "../lib/candidates";
-import employers from "../lib/employers";
+import { admins } from "../lib/admins";
+import { candidates } from "../lib/candidates";
+import { employers } from "../lib/employers";
+import Error from "./_error";
 
 const IndexPage: React.SFC<{
-    candidateName: string;
+    candidateName: string | undefined;
     role: string;
     token: string;
     username: string;
     user_id: number;
 }> = ({ candidateName, role, token, username, user_id }) => {
+    if (!token) {
```

Comment:
I really don't like generic "!" be explicit, is it undefined or null or both, or false or ""